diff --git a/RKBOOT/RK3576MINIALL.ini b/RKBOOT/RK3576MINIALL.ini
index b1f8213..af4cc49 100644
--- a/RKBOOT/RK3576MINIALL.ini
+++ b/RKBOOT/RK3576MINIALL.ini
@@ -5,7 +5,7 @@ MAJOR=1
 MINOR=100
 [CODE471_OPTION]
 NUM=1
-Path1=bin/rk35/rk3576_ddr_lp4_1848MHz_lp5_2736MHz_v1.09.bin
+Path1=bin/rk35/rk3576_ddr_lp4_2112MHz_lp5_2736MHz_v1.09.bin
 Sleep=1
 [CODE472_OPTION]
 NUM=1
@@ -16,7 +16,7 @@ LOADER1=FlashBoost
 LOADER2=FlashData
 LOADER3=FlashBoot
 FlashBoost=bin/rk35/rk3576_boost_v1.02.bin
-FlashData=bin/rk35/rk3576_ddr_lp4_1848MHz_lp5_2736MHz_v1.09.bin
+FlashData=bin/rk35/rk3576_ddr_lp4_2112MHz_lp5_2736MHz_v1.09.bin
 FlashBoot=bin/rk35/rk3576_spl_v1.07.bin
 [LOADER1_PARAM]
 LOAD_ADDR=0x3FFC0000
